src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,47 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
+    }
```

Comment:
When you convert `ResultSet` to `Book` better create an object using setters or constructor but not both of them, because it's not consistent to use both ways of initialization of object. Since you have setters for all fields, consider removing the constructor that takes parameters and use setters instead, or add an `id` parameter to the constructor if you want to initialize all fields at once.